bin/administration

```diff
@@ -307,7 +338,13 @@ EOT, PHP_EOL;
 
         $class = 'PrivateBin\\Data\\' . $this->_conf->getKey('class', 'model');
         $this->_store = new $class($this->_conf->getSection('model_options'));
-
+		
+		if ($this->_option('l', 'list-ids') !== null) {
+			$this->_list_ids();
+		}
+		if ($this->_option(null, 'delete-all') !== null) {
+			$this->_delete_all();
+		}
```

Comment:
Spaces instead of tabs:
```suggestion

        if ($this->_option('l', 'list-ids') !== null) {
            $this->_list_ids();
        }
        if ($this->_option(null, 'delete-all') !== null) {
            $this->_delete_all();
        }
```